### PR TITLE
Add parser for the bluetoothctl utility

### PR DESCRIPTION
**Pull request by tzeikob**
This is a parser proposal for the bluetoothctl utility as it's stated in the issue #391

### CODE REVIEW

**Comment by kellyjonbrazil** [comment]
Looks nice, thanks! I'll take a closer look over the weekend.

File: jc/parsers/bluetoothctl.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+        quiet:       (boolean) suppress warning messages if True
+
+    Returns:
+
+        List of Dictionaries. Raw or processed structured data.
+    """
+    jc.utils.compatibility(__name__, info.compatible, quiet)
+    jc.utils.input_type_check(data)
+
+    linedata = data.splitlines()
```
Comment (kellyjonbrazil) [inline jc/parsers/bluetoothctl.py:348]: Maybe move these two under the `if` statement? Not a big deal but no reason to process the data if the data check fails.

**Review by kellyjonbrazil — COMMENTED** [review]
Looking good - some minor suggestions.

File: jc/parsers/bluetoothctl.py [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+            "uuids": [
+                "Advanced Audio Distribu.. (0000120d-0000-1000-8000-00805f9b34fb)",
+                "Audio Sink                (0000130b-0000-1000-8000-00805f9b34fb)",
+                "A/V Remote Control        (0000140e-0000-1000-8000-00805f9b34fb)",
+                "A/V Remote Control Cont.. (0000150f-0000-1000-8000-00805f9b34fb)",
+                "Handsfree                 (0000161e-0000-1000-8000-00805f9b34fb)",
+                "Headset                   (00001708-0000-1000-8000-00805f9b34fb)",
+                "Headset HS                (00001831-0000-1000-8000-00805f9b34fb)"
+            ],
+            "rssi": "-52",
```
[the remark this answers is not in the document]
Comment (tzeikob) [reply jc/parsers/bluetoothctl.py:94]: Yes it does, good point!

**Comment by kellyjonbrazil** [comment]
Looks great! I'll merge into `dev` and this will make it in the next release (v1.23.2)

**Pull request merged**